packages/pharos/src/components/tabs/pharos-tab.ts

```diff
@@ -45,20 +45,25 @@ export class PharosTab extends PharosElement {
       this._focused = this.selected;
       this.setAttribute('aria-selected', this.selected ? 'true' : 'false');
       if (this.selected) {
-        const details = {
-          bubbles: true,
-          composed: true,
-        };
-        this.dispatchEvent(new CustomEvent('pharos-tab-selected', details));
+        this._triggerSelectedEvent();
       }
     }
+
     if (changedProperties.has('_focused')) {
       this.setAttribute('tabindex', this._focused ? '0' : '-1');
     }
   }
 
+  private _triggerSelectedEvent() {
+    const details = {
+      bubbles: true,
+      composed: true,
+    };
+    this.dispatchEvent(new CustomEvent('pharos-tab-selected', details));
+  }
+
   private _handleClick(): void {
-    this.selected = true;
+    this._triggerSelectedEvent();
   }
```

Comment:
On click, individual tabs now **only** fire the event saying that they got selected; they don't manage the transition to actual `selected` state themselves.